app/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
```

Comment:
Only for clarification: this is not a requirement of any HTML validator right? both are equivalent (and lowercase is always the easiest: no question about case)